flexmeasures/data/scripts/clean_database.sh

```diff
@@ -13,6 +13,14 @@ function is_database() {
 function create_database() {
  echo "Creating a new database ..."
  sudo -i -u postgres createdb -U postgres $1
+ if [[ -n "$2" ]]; then
+   echo "Creating database user ..."
+   sudo -i -u postgres createuser --pwprompt -U postgres $2
```

Comment:
I would only actually create this user if it does not yet exist.
Maybe a function `is_user` similar to the one you already made (`is_database`) can be ceated.